src/Application/Actions/Donations/Update.php

```diff
@@ -164,7 +164,7 @@ protected function action(Request $request, Response $response, array $args): Re
                     $this->entityManager->rollback();
 
                     // Log a warning to more easily spot occurrences in dashboards.
-                    $methodSummary = $donation->getPaymentMethodType()?->value ?? '[null]';
+                    $methodSummary = $donation->getPaymentMethodType()->value ?? '[null]';
```

Comment:
(no need for null-safe navigation to the left of `??`, as that does the same thing)